### PR TITLE
Anzhela khamitskaya feature currency rates

### PR DESCRIPTION
**Pull request by AnzhelaKhamitskaya**

### CODE REVIEW

File: src/main/java/org/dailyinfo/telegrambot/telegrambot/service/currencyRates/Rate.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+import java.util.Objects;
+
+public class Rate {
+
+    private int Cur_ID;
+    private LocalDateTime Date;
+    private String Cur_Abbreviation;
+    private int Cur_Scale;
+    private String Cur_Name;
+    private Double Cur_OfficialRate;
```
Comment (Basist92) [inline src/main/java/org/dailyinfo/telegrambot/telegrambot/service/currencyRates/Rate.java:13]: lowerCase

File: src/main/java/org/dailyinfo/telegrambot/telegrambot/service/currencyRates/Rate.java [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+    public Rate(int cur_ID, LocalDateTime date, String cur_Abbreviation, int cur_Scale, String cur_Name, Double cur_OfficialRate) {
+        Cur_ID = cur_ID;
+        Date = date;
+        Cur_Abbreviation = cur_Abbreviation;
+        Cur_Scale = cur_Scale;
+        Cur_Name = cur_Name;
+        Cur_OfficialRate = cur_OfficialRate;
```
Comment (Basist92) [inline src/main/java/org/dailyinfo/telegrambot/telegrambot/service/currencyRates/Rate.java:21]: this.var = var

File: src/main/java/org/dailyinfo/telegrambot/telegrambot/service/currencyRates/Rate.java [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    }
+
+    public int getCur_ID() {
+        return Cur_ID;
+    }
+
+    public void setCur_ID(int cur_ID) {
```
Comment (Basist92) [inline src/main/java/org/dailyinfo/telegrambot/telegrambot/service/currencyRates/Rate.java:28]: you didn`t need setter

File: src/main/java/org/dailyinfo/telegrambot/telegrambot/service/currencyRates/Rate.java [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+        Cur_ID = cur_ID;
+    }
+
+    public LocalDateTime getDate() {
+        return Date;
+    }
+
+    public void setDate(LocalDateTime date) {
```
Comment (Basist92) [inline src/main/java/org/dailyinfo/telegrambot/telegrambot/service/currencyRates/Rate.java:36]: see above

File: src/main/java/org/dailyinfo/telegrambot/telegrambot/service/currencyRates/Rate.java [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+        Date = date;
+    }
+
+    public String getCur_Abbreviation() {
+        return Cur_Abbreviation;
+    }
+
+    public void setCur_Abbreviation(String cur_Abbreviation) {
+        Cur_Abbreviation = cur_Abbreviation;
```
Comment (Basist92) [inline src/main/java/org/dailyinfo/telegrambot/telegrambot/service/currencyRates/Rate.java:45]: see above

File: src/main/java/org/dailyinfo/telegrambot/telegrambot/service/currencyRates/Rate.java [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+    }
+
+    public int getCur_Scale() {
+        return Cur_Scale;
+    }
+
+    public void setCur_Scale(int cur_Scale) {
```
Comment (Basist92) [inline src/main/java/org/dailyinfo/telegrambot/telegrambot/service/currencyRates/Rate.java:52]: see above

File: src/main/java/org/dailyinfo/telegrambot/telegrambot/service/currencyRates/Rate.java [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+        Cur_Scale = cur_Scale;
+    }
+
+    public String getCur_Name() {
+        return Cur_Name;
+    }
+
+    public void setCur_Name(String cur_Name) {
```
Comment (Basist92) [inline src/main/java/org/dailyinfo/telegrambot/telegrambot/service/currencyRates/Rate.java:60]: see above

File: src/main/java/org/dailyinfo/telegrambot/telegrambot/service/currencyRates/Rate.java [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+        Cur_Name = cur_Name;
+    }
+
+    public Double getCur_OfficialRate() {
+        return Cur_OfficialRate;
+    }
+
+    public void setCur_OfficialRate(Double cur_OfficialRate) {
```
Comment (Basist92) [inline src/main/java/org/dailyinfo/telegrambot/telegrambot/service/currencyRates/Rate.java:68]: see above

**Review by shpaser2 — APPROVED** [review]
Lgfm

**Pull request merged**